wordpress-mobile/WordPress-Android
# Automated Transfers

Fixes #7653. This PR adds the Automated Transfers feature. All the changes has been reviewed in #7609, #7661 & #7664 (including design, analytics and copy reviews). I've only merged `develop` into the branch after the last PR and ran the test below to make sure everything is still working as expected. It'd be great if the reviewer can do a test run as well before merging the code.

To test:
* Create a new WordPress.com site with business plan
* Add a custom domain to your site (A subdomain works as well, if you already have a domain, just map a subdomain of it)
* Make sure it’s public.
* Make sure you’re an admin.
* Go to My Site and check that Plugins is visible (if you have made a recent change in Calypso, you may need to go to site list and refresh the sites)
* Go into Plugins and tap on any plugin and try to install it
* Wait about a minute or so and then verify that “Successfully installed {plugin name}” snackbar shows up and the plugin shows up as installed.
* Go back to Plugins and make sure the site plugins section is now visible (Note that the backend will keep installing a few more plugins, so you may see more plugins show up in the next few minutes)
